lib/tests/change-federation.js

```diff
@@ -241,25 +459,142 @@ const execute = (description, newFederationConfig) => {
+            expect(retiringFederationInformation.creationTime).to.be.equal(initialActiveFederationInfo.creationTime, 'The retiring federation creation time should be the same as the initial federation creation time.');
+            expect(retiringFederationInformation.size).to.be.equal(initialFederationPublicKeys.length, 'The retiring federation size should be the same as the initial federation size.');
+
+            const retiringFederationPublicKeys = await getRetiringFederationPublicKeys(bridge);
+            expect(retiringFederationPublicKeys).to.be.deep.equal(initialFederationPublicKeys, 'The retiring federation public keys should be the same as the initial federation public keys.');
 
         });
 
+        it('should migrate utxos', async () => {
+
+            const bridgeStateBeforeMigration = await getBridgeState(rskTxHelper.getClient());
+            expect(bridgeStateBeforeMigration.activeFederationUtxos.length).to.be.equal(0, 'The new federation is now the active one and it should not have any utxos yet.');
+            expect(bridgeStateBeforeMigration.pegoutsWaitingForConfirmations.length).to.be.equal(0, 'No pegout should be waiting for confirmations.');
+            expect(bridgeStateBeforeMigration.pegoutsWaitingForSignatures.length).to.be.equal(0, 'No pegout should be waiting for signatures.');
+    
+            // Mining to activate the migration age
+            await rskTxHelper.mine(FUNDS_MIGRATION_AGE_SINCE_ACTIVATION_BEGIN + 1);
+
+            // Start migration
+            await rskUtils.waitAndUpdateBridge(rskTxHelper);
+            const bridgeStateAfterUpdatingCollections = await getBridgeState(rskTxHelper.getClient());
+            
+            const migrationPegouts = bridgeStateAfterUpdatingCollections.pegoutsWaitingForConfirmations;
+
+            expect(bridgeStateAfterUpdatingCollections.pegoutsWaitingForConfirmations.length).to.be.greaterThan(0, 'There should be at least one pegout waiting for confirmations.');
+    
+            await wait(1000);
+            await rskTxHelper.mine(BTC_TO_RSK_MINIMUM_CONFIRMATIONS);
+            await wait(1000);
+            await rskUtils.waitAndUpdateBridge(rskTxHelper);
+
+            const method = async () => {
+                const currentBridgeState = await getBridgeState(rskTxHelper.getClient());
+                if(currentBridgeState.pegoutsWaitingForSignatures.length === 0 && currentBridgeState.pegoutsWaitingForConfirmations.length === 0) {
+                  return true;
+                }
+                await wait(1000);
+                await rskUtils.mineAndSync(rskTxHelpers);
+                return false;
+            };
+              
+            const { result: wasPegoutBroadcasted, attempts } = await retryWithCheck(method, pegoutIsBroadcasted => pegoutIsBroadcasted, 20, 2000);
```

Comment:
For some reason, during the second federation change (svp fed change), the feds are getting delayed to sign the migration pegouts.